packages/fast-usdc/test/fast-usdc.contract.test.ts

```diff
@@ -510,115 +542,460 @@ test('LP repay', async t => {
+  // fulfilled" in the test output.
 
-  // advance sent from PoolSeat to PoolAccount
+  const { calculateAdvance, calculateSplit } = makeFeeTools(feeConfig);
+  const advance = calculateAdvance(usdc.make(sent1.tx.amount));
+  t.log('advancer sent to PoolAccount', advance);
   t.deepEqual(inspectBankBridge().at(-1), {
-    amount: String(expectedAdvance.value),
-    denom: 'ibc/usdconagoric',
+    amount: String(advance.value),
+    denom: uusdcOnAgoric,
     recipient: 'agoric1fakeLCAAddress',
     type: 'VBANK_GIVE',
   });
 
-  // ibc transfer sent over localChain bridge
-  const localBridgeMsg = inspectLocalBridge().at(-1);
-  const ibcTransferMsg = localBridgeMsg.messages[0];
-  t.is(ibcTransferMsg['@type'], '/ibc.applications.transfer.v1.MsgTransfer');
-
-  const expectedReceiver = addressTools.getQueryParams(
-    evidence.aux.recipientAddress,
-  ).EUD;
-  t.is(ibcTransferMsg.receiver, expectedReceiver, 'sent to correct address');
-  t.deepEqual(ibcTransferMsg.token, {
-    amount: String(expectedAdvance.value),
-    denom: 'ibc/usdconagoric',
-  });
+  cust1.checkSent(t, since(bridgePos));
 
-  // TODO #10445 expect PFM memo
-  t.is(ibcTransferMsg.memo, '', 'TODO expecting PFM memo');
+  const emptyMetrics = {
+    encumberedBalance: usdc.makeEmpty(),
+    shareWorth: {
+      numerator: usdc.make(1n),
+      denominator: { value: 1n },
+    },
+    totalBorrows: usdc.makeEmpty(),
+    totalContractFees: usdc.makeEmpty(),
+    totalPoolFees: usdc.makeEmpty(),
+    totalRepays: usdc.makeEmpty(),
+  };
+  const par250 = {
+    numerator: usdc.make(250_000_001n),
+    denominator: { value: 250_000_001n },
+  };
 
-  // TODO #10445 expect routing through noble, not osmosis
-  t.is(
-    ibcTransferMsg.sourceChannel,
-    fetchedChainInfo.agoric.connections['osmosis-1'].transferChannel.channelId,
-    'TODO expecting routing through Noble',
+  t.like(
+    await E(metricsSub)
+      .getUpdateSince()
+      .then(r => r.value),
+    {
+      ...emptyMetrics,
+      encumberedBalance: advance,
+      shareWorth: par250,
+      totalBorrows: advance,
+    },
+    'metrics while advancing',
   );
 
+  await mint(sent1);
+
+  // C8 - "Contract MUST be able to initialize settlement process when Noble mints USDC."
+  // The metrics are a useful proxy, but the contract could lie.
+  // The real test of whether the contract turns minted funds into liquidity is
+  // the ability to advance the funds (in later tests).
+  const split = calculateSplit(usdc.make(sent1.tx.amount));
+  t.like(
+    await E(metricsSub)
+      .getUpdateSince()
+      .then(r => r.value),
+    {
+      ...emptyMetrics,
+      shareWorth: {
+        ...par250,
+        numerator: add(par250.numerator, split.PoolFee),
+      },
+      totalBorrows: { value: 105839999n },
+      totalContractFees: { value: 432000n },
+      totalPoolFees: { value: 1728001n },
+      totalRepays: { value: 105839999n },
+    },
+    'metrics after advancing',
+  );
+});
```

Comment:
Worth a todo for checking `evmHash: status.ADVANCING` in vstorage?

Reply:
It's no longer ADVANCING at this particular point.

I'm not trying to be exhaustive about vstorage TODOs. The scope of this PR is M1. vstorage stuff is in M2.